examples/basic/lines/arcs.py

```diff
@@ -1,3 +1,20 @@
+''' A ac graph using pre-defined data points. This example
+demonstrates the use of the ``arc`` method to make a graph
+by drawing three arcs of defined radius, defined start and 
+end angles at defined point(coordinates) .
+
+
+
+.. bokeh-example-metadata::
```

Comment:
Excess vertical space
```suggestion
end angles at defined point(coordinates) .

.. bokeh-example-metadata::
```